packages/components/package.json

```diff
@@ -37,6 +37,7 @@
     "ci:test:react-17": "USE_REACT_17=true yarn test:ci"
   },
   "dependencies": {
+    "@cultureamp/fetch": "^3.0.9",
```

Comment:
A peer dep for `@cultureamp/i18n-react-intl`

Reply:
Which... actually isn't, so KAIO was complaining for nothing...

Reply:
What if you peerDep the peerDep 😛